exercises/concept/emoji-times/.docs/hints.md

```diff
@@ -1,4 +1,4 @@
-## 3. Define emoji constants to represent instances of `Clock`
+# 3. Define emoji constants to represent instances of `Clock`
```

Comment:
```suggestion
# Hints

## 3. Define emoji constants to represent instances of `Clock`
```

?

Reply:
Nice find, I completely missed that. Fixed in f3589ab0cc8988bc0606de28b74126c192c95b22